docs/elasticsearch.md

```diff
@@ -14,195 +14,487 @@ sidebar_label: Elasticsearch
-- If required, take backup of the Elasticsearch database periodically and back them up to S3 or any object storage so that restoration of the same logs is possible to the same or any other Kubernetes cluster
+- Low latency and better performance
 
 <br>
 
-*Note: Elasticsearch can be deployed both as `deployment` or as `statefulset`. When Elasticsearch deployed as `statefulset`, you don't need to replicate the data again at OpenEBS level. When Elasticsearch is deployed as `deployment`, consider 3 OpenEBS replicas, choose the StorageClass accordingly.*
-
-<br>
 
-<hr>
+## Deployment model
 
 <br>
+<img src="/docs/assets/svg/Local-PV-Devices-elastic-deployment.svg" alt="OpenEBS and Elasticsearch" style="width:100%;">
 
+The Local PV volume will be provisioned on a node where Elasticsearch components are getting scheduled and uses one of the matching unclaimed block device for each of them, which will then use the entire block device for storing data. No other application can use this device. If users have limited blockdevices attached to some nodes, they can use `nodeSelector` in the application YAML to provision applications on particular nodes where the available block device is present. The recommended configuration is to have at least three nodes and two unclaimed external disk to be attached per node. 
 
+The Elasticsearch deployment has the following components, which will use the OpenEBS LocalPV Devices for storage:
+1. coordinator pod: 1
+2. master pods: 3
+3. data pods: 2 
 
-## Deployment model
 
+*Note: Elasticsearch can be deployed both as `deployment` or as `statefulset`. When Elasticsearch deployed as `statefulset`, you don't need to replicate the data again at OpenEBS level. When Elasticsearch is deployed as `deployment`, consider 3 OpenEBS replicas, choose the StorageClass accordingly.*
 <br>
 
+## Configuration workflow
 
+1. [Install OpenEBS](/docs/next/elasticsearch.html#install-openebs)
+2. [Select OpenEBS storage engine](/docs/next/elasticsearch.html#select-openebs-storage-engine)
+3. [Configure OpenEBS Local PV StorageClass](/docs/next/elasticsearch.html#configure-openebs-local-pv-storageclass)
+4. [Installing KUDO Operator](/docs/next/elasticsearch.html#installing-kudo-operator)
+5. [Installing and Accessing Elasticsearch](/docs/next/elasticsearch.html#installing-and-accessing-elasticsearch)
+6. [Installing Kibana](/docs/next/elasticsearch.html#installing-kibana)
+7. [Installing Fluentd-ES](/docs/next/elasticsearch.html#installing-fluentd-es)
 
-<img src="/docs/assets/svg/elasticsearch-deployment.svg" alt="OpenEBS and Elasticsearch" style="width:100%;">
 
-<br>
 
-<hr>
+### Install OpenEBS
 
-<br>
+If OpenEBS is not installed in your K8s cluster, this can be done from [here](/docs/next/installation.html). If OpenEBS is already installed, go to the next step. 
 
-## Configuration workflow
 
-<br>
+### Select OpenEBS storage engine
```

Comment:
You may need to update this section as per done for Prometheus. https://docs.openebs.io/docs/next/prometheus.html#select-openebs-storage-engine
You can point out the link once the PR is approved in cStor operator repo.